src/main/java/org/takes/tk/TkWrap.java

```diff
@@ -39,6 +39,7 @@
  * @version $Id$
  * @since 0.4
  */
+@SuppressWarnings("PMD.AvoidDuplicateLiterals")
```

Comment:
@dalifreire if this one has duplicate literals problems, we should add it to the exclude list in `pom.xml` so the dev that gets the puzzle can work on it also.